src/controllers/mainController.ts

```diff
@@ -410,6 +416,14 @@ export default class MainController implements vscode.Disposable {
     }
 
     /**
+     * Verifies the tools service version is high enough to support certain commands
+     */
+    private CanRunV2Command(): boolean {
```

Comment:
I just noticed @MattIrv  renamed canRunCommand in his active PR, so you don't need to rename that one